igvm/commands.py

```diff
@@ -183,6 +185,43 @@ def disk_set(vm_hostname, size):
         vm.dataset_obj.commit()
 
 
+@with_fabric_settings
+def change_address(vm_hostname, new_address):
+    """Change VMs IP address
+
+    This is done by changing data in Serveradmin, running Puppet in VM and
+    rebooting it.
+    """
+
+    with _get_vm(vm_hostname) as vm:
+        old_address = vm.dataset_obj['intern_ip']
+        new_address = ip_address(new_address)
+
+        if old_address == new_address:
+            raise ConfigError('New IP address is the same as the old one!')
+
+        vm_was_running = vm.is_running()
+
+        vm.dataset_obj['intern_ip'] = new_address
+        vm.dataset_obj.commit()
+
+        if vm_was_running:
```

Comment:
Maybe it is a good idea to at least write in the comment that on IP change the VM gets rebootet.

Reply:
I thought of something else: force this command to require `--offline`. That would make it consistent with other commands we have.

Reply:
This is fine too.